src/sentry/integrations/pagerduty/integration.py

```diff
@@ -14,12 +14,7 @@
     IntegrationProvider,
     FeatureDescription,
 )
-from sentry.models import (
-    OrganizationIntegration,
-    PagerDutyServiceProject,
```

Comment:
Do these models need to be removed?

Reply:
@manuzope is going to delete the `PagerDutyServiceProject` model in a subsequent PR